packages/storybook/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
```

Comment:
do we need to have `storybook` under packages or can live in root?

Reply:
It can be outside of `packages/`. That's how it's set up in https://github.com/backstage/backstage/tree/master/storybook

Reply:
TL;DR I think for simplicity it's better to have it inside `packages/` as a private package